src/component/utils/getElementForBlockType.js

```diff
@@ -17,9 +17,15 @@ import type {DraftBlockType} from 'DraftBlockType';
 function getElementForBlockType(blockType: DraftBlockType): string {
   switch (blockType) {
     case 'header-one':
-      return 'h2';
```

Comment:
This was intended, but is a decision from long ago. I'll check to see if we can go ahead and change it. Your change is surely more sane for a public repo. :)